integration-tests/check-subscriptions

```diff
@@ -109,6 +111,22 @@ def machine_python(machine, script, arg=""):
     return machine.execute("python3 -c %s %s" % (shell_escape(script), shell_escape(arg)))
 
 
+def skipUnlessDistroFamily(distro, reason):
+    """
+    Skip the current test function with the specified [reason]
+    unless the distribution of the machine is part of the specified
+    family [distro] (i.e. it starts as "distro-").
+    """
+    def distro_wrapper(func):
+        @functools.wraps(func)
+        def wrapper(self):
```

Comment:
Using `self` here is little bit confusing, because `self` is used in methods of class. This is not this case. Can you find better name like `obj` to emphasize the difference.